velox/dwio/common/IntCodecCommon.h

```diff
@@ -43,4 +44,9 @@ constexpr int64_t MAX_NANOS = 999'999'999;
 // 1 is reduced to epoch, as writer adds 1 for negative seconds.
 constexpr int64_t MIN_SECONDS = INT64_MIN + (EPOCH_OFFSET - 1);
 
+#if not __has_builtin(__builtin_bswap128)
+#define __builtin_bswap128(X) \
+  buildInt128(__builtin_bswap64(X), __builtin_bswap64(X >> 64))
```

Comment:
Put extra parenthesis to avoid operator precedence issue: `buildInt128(__builtin_bswap64((X)), __builtin_bswap64((X) >> 64))`